synapse/handlers/presence.py

```diff
@@ -1141,6 +1143,33 @@ def _push_presence_remote(self, user, destination, state=None):
             content={"push": [user_state, ], }
         )
 
+    def get_all_presence_updates(self, last_id, current_id, limit):
+        """Return all the presence updates happened since the last id"""
+        # TODO: Respect the limit.
```

Comment:
Eh, I don't think the method without a limit would be useful to anyone. I'd rather that the parameter stayed and the method was fixed.